Doesn't compile on many gcc versions

After adding the AES stuff, it fails to compile on many gcc versions:

```
cc -fPIC -std=c99 -O3 -mavx2  -march=native -Wall -Wextra -pedantic -Wshadow -o fillarray ./benchmark/fillarray.c simdxorshift128plus.o xorshift128plus.o -Iinclude -flto
In file included from ./benchmark/fillarray.c:7:0:
include/simdaesdragontamer.h: In function ‘aesdragontamer_r’:
include/simdaesdragontamer.h:25:10: warning: implicit declaration of function ‘_mm256_set_m128i’ [-Wimplicit-function-declaration]
   return _mm256_set_m128i(penultimate1,penultimate2);
          ^
include/simdaesdragontamer.h:25:10: error: incompatible types when returning type ‘int’ but ‘__m256i {aka __vector(4) long long int}’ was expected
include/simdaesdragontamer.h:26:1: warning: control reaches end of non-void function [-Wreturn-type]
 }
 ^
```

It is because `_mm256_set_m128i` doesn't exist on gcc until recently.

--- a/include/simdaesdragontamer.h
+++ b/include/simdaesdragontamer.h
@@ -22,7 +22,7 @@ static inline __m256i aesdragontamer_r(aesdragontamer_state *state) {
   __m128i penultimate = _mm_aesenc_si128(state->state, state->increment); 
   __m128i penultimate1 = _mm_aesenc_si128(penultimate, state->increment); 
   __m128i penultimate2 = _mm_aesdec_si128(penultimate, state->increment); 
-  return _mm256_set_m128i(penultimate1,penultimate2);
+  return _mm256_insertf128_si256(_mm256_castsi128_si256(penultimate2), penultimate1, 1);
 }
 
 

